python/ray/tune/suggest/skopt.py

```diff
@@ -91,6 +98,13 @@ def __init__(self,
         assert type(max_concurrent) is int and max_concurrent > 0
         _validate_warmstart(parameter_names, points_to_evaluate,
                             evaluated_rewards)
+        assert mode in ["min", "max"], "mode must be 'min' or 'max'!"
+
+        if reward_attr is not None:
+            mode = "min"
+            metric = reward_attr
+            logger.warning("`reward_attr` will be depreciated!"
+                           "Consider using `metric` and `mode`.")
```

Comment:
same as above. This file should have `mode=max` and `metric=episode_reward_mean`. I think you flipped the `min` and `max` because of the negative value in skopt.tell? But understand that the user will set `episode_reward_mean` and `max` and expect it to be maximized.